nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -27,12 +28,43 @@ class GraphConverter:
     Builds the NNCFGraph from an torch.fx.GraphModule instance.
     """
 
+    def _get_node_subtype(
+        node: torch.fx.Node, metatype: om.OperatorMetatype, model: torch.fx.GraphModule
+    ) -> om.OperatorMetatype:
+        """
+        Attempts to retrieve correct subtype for the given node.
+
+        :param node: Given node.
+        :param metatype: Given node metatype.
+        :param model: Target GraphModule instance.
+        :return: Correct subtype of the given node if it is exist or the original node metatype otherwise.
+        """
+        if metatype in [om.PTConv1dMetatype, om.PTConv2dMetatype, om.PTConv3dMetatype]:
+            if len(node.args) < 7:
+                return metatype
+            constant_node = node.args[1]
```

Comment:
constant_node is it weight or bias?
Please use more correct name or add comment.

Reply:
Comment is added, please check

Reply:
Outdated